tests/FileHelperTest.php

```diff
@@ -1,19 +1,30 @@
 <?php
+declare(strict_types = 1);
+
 namespace Yiisoft\Files\Tests;
 
 use PHPUnit\Framework\TestCase;
 use Yiisoft\Files\FileHelper;
 
+/**
+ * File helper tests class.
+ */
 final class FileHelperTest extends TestCase
 {
     /**
      * @var string test files path.
      */
     private $testFilePath = '';
 
-    public function setUp()
+    /**
+     * Setup.
```

Comment:
Such phpdoc is not useful so could be removed.

Reply:
Done.